src/applications/hca/containers/IdentityPage.jsx

```diff
@@ -72,6 +72,7 @@ const IdentityPage = props => {
       } = localData;
       setFormData({
         ...formData,
+        veteranDateOfBirth,
```

Comment:
While we don't use this value for guest users in the submit transformer--because it gets stripped out--we need to include it for the service date validation, since authenticated user data has this key populated. We handle guest user data with the `view:veteranInformation` object down on L78.